app/helpers/submissions_helper.rb

```diff
@@ -87,10 +87,14 @@ def formatted_minimum_price_for_email(submission)
   end
 
   def note_byline(note)
```

Comment:
I'm late with this feedback, but I would not mind this method moving to the Note class.